readme.md

```diff
@@ -537,6 +525,50 @@ const Demo = (
 );
 ```
 
+Ink also supports Fragments for returning multiple children from a component's render method.
+
+```jsx
+render(
+	<h.Fragment>
```

Comment:
Could you export `Fragment` on the root export instead of `h`, so that it becomes:

```jsx
import { render, Fragment } from 'ink';

<Fragment>
	<A/>
	<B/>
	<C/>
</Fragment>
```